packages/vkui/src/components/PopoutWrapper/PopoutWrapper.tsx

```diff
@@ -18,17 +18,34 @@ const stylesAlignY = {
   bottom: styles.alignYBottom,
 };
 
+const stylesStrategy = {
+  fixed: styles.fixed,
+  absolute: styles.absolute,
+  none: undefined,
+};
+
 export interface PopoutWrapperProps extends HTMLAttributesWithRootRef<HTMLDivElement> {
   /**
    * Позволяет сделать прозрачную подложку
    */
   noBackground?: boolean;
   /**
+   * @deprecated будет удалён в **VKUI v8**
+   * Используйте `strategy` вместо этого свойства.
+   *
    * Включает фиксированное позиционирование.
    *
-   * По умолчанию у компонента не задан никакой `position`.
+   * При значении `false` у компонента не задан никакой `position`.
    */
   fixed?: boolean;
+  /**
+   * Стратегия позиционирования:
+   *
+   * - `fixed`: у контейнера выставлен `position: fixed`
+   * - `absolute`: у контейнера выставлен `position: absolute`
+   * - `none`: у контейнера не выставлен `position`
```

Comment:
```suggestion
   * - `none`: у контейнера не выставлен `position`
   *
   * @default 'fixed'
```

так как в 75-ой строчке пока нет возможности показать, что параметр по умолчанию это `fixed`